client/src/common/codeAction.ts

```diff
@@ -53,9 +53,11 @@ export class CodeActionFeature extends TextDocumentLanguageFeature<boolean | Cod
 					CodeActionKind.Refactor,
 					CodeActionKind.RefactorExtract,
 					CodeActionKind.RefactorInline,
+					CodeActionKind.RefactorMove,
```

Comment:
Also adds `RefactorMove` in addition to the stated (in the commit message) `Notebook`?

Reply:
The `RefactorMove` got added a while back but not correctly send on initialize.